pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -27,7 +27,7 @@ import (
 )
 
 type k8sBindingInjector struct {
-	client *kubernetes.Clientset
+	client kubernetes.Interface
```

Comment:
What are the implications of changing this? I'm not familiar with Clientset or Interface.